docs/contracts/NFT Contract/NFT Actions/purchase.a.md

```diff
@@ -0,0 +1,140 @@
+---
+title: 'purchase.a'
+order: 31
+deploy: ['experimental']
+---
+
+# purchase.a
+
+This action is used to purchase uniqs directly from a token factory.
+
+## Technical Behavior
+
+1. User provides information about the Uniq they wish to purchase.
+
+2. Verify that the Uniq has a purchase requirement
+
+3. Obtain the price of the Uniq and convert to UOS
+
+4. Verify that if the purchase requirement requires additional uniqs that the uniqs passed are relevant to the purchase requirement.
+
+5. Additional transfer, and burning actions may be used on individual uniqs during the verification process. (They are kept if the transaction fails)
+
+6. Distribute shares based on purchase requirements, done through inline calls
+
+7. Send protocol fee. Amount and account are configured under `global.share` table scope `0`, done through inline calls
+
+8. Send remainder of shares to the factory manager, done through inline calls
+
+9. Issue the token to the user. **Note**: minting a Uniq requires additional UOS paid by the factory manager. Refer for details to [issue.b - issue tokens with token factory](./issue.b.md)
+
+10. Increment the number of tokens purchased for the given user
```

Comment:
This list has inconsistent punctuation. Each number should have a period, or not have period. Not both.